docs/releasing/testing-and-linting.md

```diff
@@ -47,3 +47,23 @@ If a snapshot test fails, review the difference in the console. If the change is
 `npm test -- -u src/govuk/components/button`
 
 This will update the snapshot file. Commit this file separately with a commit message that explains you're updating the snapshot file and an explanation of what caused the change.
+
+## Visual regression testing with Percy
+
+We use [Percy](https://percy.io/), a visual regression testing tool, to generate and store screenshots of our components to help us check for any unintended visual changes.
+
+We generate two screenshots for each default example of every component: one with JavaScript enabled and one with JavaScript disabled. Screenshots are not taken for all the different variations of each component. This tool is not a replacement for manual testing.
+
+The screenshots are public, so they can be checked without logging in. A BrowserStack account is needed to approve or reject any changes. It's the responsibility of the person reviewing the pull request code to approve any visual changes that Percy highlights.
+
+When running the tests locally (e.g: via `npm test`), Percy commands are ignored and no screenshots are generated. You will see the following message in your command line output: `[percy] Percy is not running, disabling snapshots`.
+
+### PRs from forks
+When Github Actions is running against a PR from a fork, the Percy secret is not available and therefore no screenshots are generated. Other tests will continue to run as normal. You will see the following messages in the output:
+
+```
+[percy] Skipping visual tests
+[percy] Error: Missing Percy token
+```
+
+This is the reason why we are unable to make Percy a required check for this repo. However, we should continue to act as if it is required. Percy runs should be approved before merging, and we should treat any failures as blocking.
```

Comment:
```suggestion
These messages are the reason we're unable to make Percy a required check for this repo. However, we should continue to act as if a Percy check is required. You should approve Percy runs before merging pull requests. Treat any failures as blocking.
```
Will it be obvious to users what action they should take in order to "treat any failures as blocking"?

Reply:
I guess what I mean there is, if the Percy build fails, don't just merge the PR because Github lets you. You should continue to review the changes and approve/reject them as necessary. There probably is a clearer way of writing that...

Reply:
Thanks for explaining! Your wording looks good - maybe we could add in something like the following:

```suggestion
These messages are the reason we're unable to make Percy a required check for this repo. However, we should continue to act as if a Percy check is required. If the Percy build fails, do not merge the pull request even though GitHub would let you. Continue to review the changes, and approve or reject them as you would normally.
```